### PR TITLE
Fix ProductImage crashing when there is no image to render

### PR DESCRIPTION
**Pull request by Klynger**
#### What is the purpose of this pull request?
As the title says

#### How should this be manually tested?
1. [workspace](https://klynger--storecomponents.myvtex.com/home---decor/?__disableSSR)
2. To test this I removed all images in code to reproduce the use case, but if you want to see a product with an image, just open the quick view of the second item and it will magically display one

#### Screenshots or example usage

image

#### Types of changes
- [X] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,10 +6,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- `ProductImage` crashing if there is no image to render.
 
 ## [2.54.0] - 2020-05-12
 ### Added
-- `href` prop in `ProductSummaryCustom` 
+- `href` prop in `ProductSummaryCustom` .
```
Comment (iaronaraujo) [inline CHANGELOG.md:14]:
```suggestion
- `href` prop in `ProductSummaryCustom`.
```

File: react/components/ProductSummaryImage/ProductImage.js [hunk 2]
```diff
@@ -155,19 +155,19 @@ const ProductImageContent = ({
     legacyContainerClasses
   )
 
-  if (!sku || hasError) {
+  const images = pathOr([], ['images'], sku)
```
Comment (klzns) [inline react/components/ProductSummaryImage/ProductImage.js:158]:
```suggestion
  const images = sku?.images ?? []
```

File: react/components/ProductSummaryImage/ProductImage.js [hunk 3]
```diff
@@ -155,19 +155,19 @@ const ProductImageContent = ({
...
+  const hoverImage = findImageByLabel(images, hoverImageLabel)
+
+  let imageUrl = pathOr('', ['image', 'imageUrl'], sku)
```
Comment (klzns) [inline react/components/ProductSummaryImage/ProductImage.js:161]:
```suggestion
  let imageUrl = sku?.image?.imageUrl ?? ''
```

Comment (Klynger) [reply react/components/ProductSummaryImage/ProductImage.js:159]: We don't support this features to `js`, so I think we should only try to make this change when we pass these files to `tsx`

Comment (Klynger) [reply react/components/ProductSummaryImage/ProductImage.js:159]: That's why I'm not removing ramda too

**Pull request merged**